### ISSUE TITLE
Screening room: needs a number of fixups

### ISSUE THREAD

**Issue by yarikoptic** [sparkletown/sparkle#1568]
- [ ] search is plain straight fuss.js without what was done for posters hall.  impossible to find anything (e.g. by author name) https://github.com/sparkletown/sparkle/issues/1448
- [ ] title for the https://ohbm.sparkle.space/in/screeningroom web page says "OHBM2021- Poster Hall" not "OHBM2021- Screening Room"
- [ ] would be a nice UX to "scroll to top" to the video pane if I hit on a preview down in the list of videos, and ideally go back to the original location when I close that video .  Otherwise it is at least confusing and requires figuring out to scroll to the top to actually see the video when "Now playing" appears overlayed on top of the preview (note: click on "Now playing" also doesn't scroll up)
- [ ] "now playing" is factually incorrect since clicking/selecting a video doesn't actually start playing it.  I think it would be a nice UX addition to start playing video as soon as it is selected

Otherwise -- I really like it! well done!

attn @margulies

**Comment by katjaq** [comment]
Yes, indeed, super nice! I would like to add to the list above that in screening room space for symposia, the cards for each symposium are very thin, and yet they expand way beyond the screen width. The right most card is well readable, the left most card is cut.

Maybe we could arrange the symposia cards in 2 rows instead of one?

Otherwise, it works very nicely ✨

**Comment by katjaq** [comment]
just saw your PR #1564  aims at fixing this ! Looks great! 🚀

**Comment by katjaq** [comment]
workaround with a bookmarklet? :D 
This is a bookmarklet to play the video in the screening room right where you are instead of scrolling up etc...
```
javascript:(function(){dqs=document.querySelector(".ScreeningRoom__video-cover").style;dqs.position="fixed";dqs.zIndex="100";dqs.backgroundColor="rgba(0,0,0,0.5)"})(); 
```
:smile:
(well, it's not  perfect... you have to click every time :upside_down: but you don't lose your position ^^)
